packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -166,29 +170,31 @@ class StdoutLogger extends Logger {
   @override
   Status startProgress(
     String message, {
+    @required Duration timeout,
     String progressId,
-    bool expectSlowOperation,
-    bool multilineOutput,
-    int progressIndicatorPadding,
+    bool multilineOutput = false,
+    int progressIndicatorPadding = kDefaultStatusPadding,
   }) {
-    expectSlowOperation ??= false;
-    progressIndicatorPadding ??= kDefaultStatusPadding;
+    assert(timeout != null);
+    assert(progressIndicatorPadding != null);
     if (_status != null) {
       // Ignore nested progresses; return a no-op status object.
-      return Status(onFinish: _clearStatus)..start();
+      return SilentStatus(
+        timeout: timeout,
+        onFinish: _clearStatus,
+      )..start();
     }
     if (terminal.supportsColor) {
       _status = AnsiStatus(
         message: message,
-        expectSlowOperation: expectSlowOperation,
-        multilineOutput: multilineOutput,
```

Comment:
I think you want to keep `multilineOutput: multilineOutput,`

Reply:
oops. done.